src/components/Camera/Video.js

```diff
@@ -1,64 +1,186 @@
 // @flow
```

Comment:
@stefaniacardenas , @rfreitas 

I think this file has grown a bit too much, and it's responsibilities are way outside the scope of simply recording a video.

It feels some of the functionality should be moved outside `/Camera`, with pure video recording kept inside `/Camera` in a `Video` component, and fetching challenges and challenge controls inside a `Liveness` component. But on the other hand `Photo` very similar controls, and almost identical presentation, so I'm not sure if decoupling it like this would keep `Video` and `Photo` consistent.

I tried a few approaches, but somehow video state, recording, controls, and challenges are very closely related, so it's something that still needs work. Your opinions and thoughts will be very welcome.

Reply:
Yes, I agree that some functionalities should be moved outside Camera. You could have a wrapper for `Video` that handles recording and challenges. I think we can refactor this logic as part of another story. What do you think @rfreitas?

Reply:
I'm pretty sure I replied to this.... Github reviews has very poor UX.

anyway. I agree, however I think the main separation has been done already between camera (react-webcam) and business logic (face, liveness, etc). I think for this component we might just have to rename it (LivenessCapture?) and move it to its own folder.

Reply:
I was reading the come more and I'm more convinced that we should move this to its own folder (or to `/Liveness`) and call is `LivenessCapture`